src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigsCollection.java

```diff
@@ -85,50 +49,52 @@ public String toPackedFasta(){
      * @param packedFastaLine
      */
     @VisibleForTesting
-    protected static ContigsCollection fromPackedFasta(final String packedFastaLine) {
+    protected static ContigsCollection fromPackedFasta(final long assemblyID, final String packedFastaLine) {
```

Comment:
Actually the individual `LocalAssemblyContig`'s will be accessed in the genotyping step, because not all contigs will be coming from the same assembly.
Does this make things a bit clearer?
In fact I've been thinking of removing `ContigsCollection` all together, but realized that would be too big a PR so just laid back and being lazy...